spec/decorators/case/ico/base_decorator_spec.rb

```diff
@@ -1,7 +1,19 @@
 require "rails_helper"
 
 describe Case::ICO::BaseDecorator do
+  let(:closed_ico_foi_case) { (create :closed_ico_foi_case).decorate }
+  let(:closed_ico_sar_case) { (create :closed_ico_sar_case).decorate }
```

Comment:
This case type isn't used here. We discussed whether this is the right place to, essentially, test `Case::ICO::FOIDecorator` and `Case::ICO::SARDecorator` ... but it's not really clear what place is the best. :( I say we take the path of least resistance and remove this `let` from here for now.